JASP-Desktop/analysisforms/Common/descriptivesform.cpp

```diff
@@ -34,6 +34,8 @@ DescriptivesForm::DescriptivesForm(QWidget *parent) :
 
 	TableModelVariablesAssigned *model = new TableModelVariablesAssigned(this);
 	model->setSource(&_availableVariablesModel);
+    model->setVariableTypesSuggested(Column::ColumnTypeNominal | Column::ColumnTypeOrdinal | Column::ColumnTypeScale);
+    model->setVariableTypesAllowed(Column::ColumnTypeNominal | Column::ColumnTypeOrdinal | Column::ColumnTypeScale | Column::ColumnTypeNominalText);
```

Comment:
We should allow any type to be added in descriptives. This doesn't change that, does it?

Reply:
There is one other type not covered by this syntax, namely type "unknown". Do you want this type to be allowed as well?

Reply:
Can you make a type that's unknown and see how JASP respond to it? I think it's a safeguard, but I'm curious and would like to see how it reacts. Otherwise, this pull request is great.

Reply:
Bruno explained to me that it is not possible to have data type unknown. The type is there for sth else.

I simplified the code by deleting the allowed line for descriptives because the default is that everything is allowed anyway.